src/lib/operators/abstract_operator.hpp

```diff
@@ -77,8 +77,13 @@ class AbstractOperator : public std::enable_shared_from_this<AbstractOperator>,
   virtual void execute();
 
   // returns the result of the operator
+  // When using OperatorTasks, they automatically clear this once all successors are done. This reduces the number of
+  // temporary tables.
   std::shared_ptr<const Table> get_output() const;
 
+  // clears the output of this operator to free up space
+  void clear_output();
```

Comment:
Add doc `DANGEROUS: Only call this if you know what you're doing, and you probably don't` or similar? This should really only be called by the OperatorTask, right?

Reply:
Hmm, I don't think that this is any more dangerous than other operations. If you clear things, things break. This doesn't look like an extraordinarily hidden pitfall to me.

Reply:
you're probably right.